docs/reference/eql/functions.asciidoc

```diff
@@ -276,4 +277,82 @@ function returns the remaining string.
 Positions are zero-indexed. Negative offsets are supported.
 
 *Returns:* string
+====
+
+[discrete]
+[[eql-fn-wildcard]]
+=== `wildcard`
+Returns `true` if a source string matches one or more provided wildcard
+expressions. Matching is case sensitive.
+
+[%collapsible]
+====
+*Example*
+[source,eql]
+----
+// The two following expressions are equivalent.
+process.command_line == "*start*" or process.command_line == "*config*"
+wildcard(process.command_line, "*start*", "*config*")
+
+// process.command_line = "start explorer.exe"
+wildcard(process.command_line, "*start*")                // returns true
+wildcard(process.command_line, "*start*", "*config*")    // returns true
+wildcard(process.command_line, "*config*")               // returns false
+wildcard(process.command_line, "*config*", "*create*")   // returns false
+
+// process.command_line = [ "start explorer.exe", "create regsvr32.exe" ]
+wildcard(process.command_line, "*create*")              // returns true
+wildcard(process.command_line, "*create*", "*config*")  // returns true
+wildcard(process.command_line, "*start*")               // returns false
+wildcard(process.command_line, "*start*", "*create*")   // returns true
+wildcard(process.command_line, "*start*", "*config*")   // returns false
+
+// case sensitive matching
+wildcard("start explorer.exe", "*start*")                // returns true
+wildcard("start explorer.exe", "*Start*")                // returns false
+
+// null handling
+wildcard(null, "*start*")                                // returns false
+wildcard(null, null)                                     // returns 500 error
+wildcard("start explorer.exe", null)                     // returns 500 error
+wildcard("start explorer.exe", null, "*start*")          // returns 500 error
+wildcard("", null)                                       // returns 500 error
```

Comment:
This behavior isn't consistent with other string functions, like `startsWith`: https://www.elastic.co/guide/en/elasticsearch/reference/master/eql-function-ref.html#eql-fn-startswith

I wonder if it would be more consistent to just pass a `null` value?

Reply:
With `wildcard` all parameters after the first must be static strings, so these will get verification failures.

Reply:
Makes sense.